src/wallet/wallet_rpc_server.cpp

```diff
@@ -1698,6 +1702,7 @@ namespace tools
       rpc_payment.amount       = payment.m_amount;
       rpc_payment.block_height = payment.m_block_height;
       rpc_payment.unlock_time  = payment.m_unlock_time;
+      rpc_payment.locked       = !m_wallet->is_transfer_unlocked(payment.m_unlock_time, payment.m_block_height);
```

Comment:
The same should be done for the two places in `on_get_bulk_payments` where `wallet_rpc::payment_details` gets filled.